src/redux/books/books.js

```diff
@@ -0,0 +1,36 @@
+const ADD_BOOK = 'BOOK_ADDED';
+const REMOVE_BOOK = 'BOOK_REMOVED';
```

Comment:
- [ ] These action types need to follow the duck's pattern, according to the project requirements:

![image](https://user-images.githubusercontent.com/57421823/166410186-2c3225d6-4dc0-4db7-baf9-cbf0a7e2ba45.png)


Hint:  Examples of duck's pattern would be: 

```javascript
const ADD_BOOK = bookstore/books/ADD_BOOK 
const REMOVE_BOOK = bookstore/books/REMOVE_BOOK
```

Reply:
@Meltrust  thank you for that. I have added it see below: 
![e59](https://user-images.githubusercontent.com/40554384/166630076-962d059c-d8b4-43ba-8938-d7e5f7ed52bc.PNG)